charts/tfm-almacar/charts/dal/templates/configmap_properties.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ include "dal.fullname" . }}
+data:
+  application.yaml: |-
+    grpc-server:
+      port: {{ .Values.global.dal.port }}
+      test: false
+
+    spring:
+      datasource:
+        driver-class-name: com.mysql.cj.jdbc.Driver
+        url: jdbc:mysql://{{ .Values.global.mysql.host }}:{{ .Values.global.mysql.port }}/mybbdd
```

Comment:
I think database name should be a variable too.

Reply:
I was thinking about that, and username and userpass, so finally I kept them as hardcoded/internal for application. But I would not mind to do it.

Reply:
We have agreed to make configurable all mysql parameters (rootPass, database, username & user password).
New issue will be open to handle those changes. And new PR will be created for that purpose.
So I proceed to upload this PR.